File: portal_backend/src/entities/agile_system_user.ts
import {
  Entity,
  PrimaryGeneratedColumn,
  Column,
  ManyToOne,
  JoinColumn
} from 'typeorm'
import { AgileSystem } from '../entities/agile_system'
import { WorkTeamMember } from '../entities/work_team_member'
@Entity({ name: 'agile_system_user' })
export class AgileSystemUser {
  @PrimaryGeneratedColumn('uuid', { name: 'agile_system_user_id' })
  agileSystemUserId: string

  @Column('varchar', { name: 'agile_system_user_name' })
  agileSystemUserName: string

  @ManyToOne(type => AgileSystem)
  @JoinColumn({ name: 'agile_system_id' })
  agileSystem: AgileSystem

  @ManyToOne(type => WorkTeamMember)
  @JoinColumn({ name: 'work_team_member_id' })
  workTeamMember: WorkTeamMember

  @Column('tinyint', { name: 'active' })
  active: boolean

}

File: fg_metrics/VSTS/VSTSComms/VSTSComms/EntitiesMySQL/AgileSystemUser.cs
using System;
using System.Collections.Generic;

namespace VSTSComms.EntitiesMySQL
{
    public partial class AgileSystemUser
    {
        public AgileSystemUser()
        {
            AgileStoryAgileSystemUser = new HashSet<AgileStoryAgileSystemUser>();
        }

        public string AgileSystemUserId { get; set; }
        public string AgileSystemUserName { get; set; }
        public string AgileSystemId { get; set; }
        public string WorkTeamMemberId { get; set; }
        public bool? Active { get; set; }

        public WorkTeamMember WorkTeamMember { get; set; }
        public ICollection<AgileStoryAgileSystemUser> AgileStoryAgileSystemUser { get; set; }
    }
}
